src/colors.config.ts

```diff
@@ -3,18 +3,21 @@ const colors = {
     // Background colors in capacitor app needs to be in hexadecimal codes
     bg: '#000000',
     bgOverlay80: 'rgba(0, 0, 0, 0.8)',
+    bgOverlay50: 'rgba(0, 0, 0, 0.5)',
+    bgOverlay30: 'rgba(0, 0, 0, 0.3)',
     black: 'rgba(0, 0, 0, 1)',
     blue: 'rgba(0, 199, 230, 1)', // #00c7e6
     darkgray: 'rgba(17, 17, 17, 1)', // #111111
     fg85: 'rgba(217, 217, 217, 1)', // #d9d9d9
     fg: 'rgba(255, 255, 255, 1)',
     fgOverlay80: 'rgba(20, 20, 20, 0.8)',
     fgOverlay90: 'rgba(20, 20, 20, 0.9)',
+    fgOverlay50: 'rgba(255, 255, 255, 0.5)',
     gray15: 'rgba(38, 38, 38, 1)', // #262626
     gray33: 'rgba(85, 85, 85, 1)',
     gray50: 'rgba(128, 128, 128, 1)', // #808080 (gray)
     gray66: 'rgba(169, 169, 169, 1)', // #a9a9a9
-    gray: 'rgba(169, 169, 169, 1)', // #a9a9a9
+    gray: 'rgba(169, 169, 169, 1)', // #a9a9a9, this is used for disabled things + text color, unlike gray66 it is the same for both light and dark
```

Comment:
Panda tokens have support for both a `value` and `description`. It would require some refactoring (`convertColorsToPandaCSS`), but do you think it would be worth considering using the `description` instead of comments for notes like this one?

This is just a question, not a blocking issue.

Reply:
I think this would be worth doing, so also moving things so it is:
```js
gray: {
  dark: ...,
  light: ...,
  description: ...,
}
```
I can refactor this in a future PR @raineorshine if you'd like

Reply:
I'm not sure. PandaCSS descriptions are nice in theory, but do not have any editor support as far as I know. JSDOC seems more reliable and widely supported.